### DISCUSSION TITLE
Потеря связи с БД при большом трафике данных (максимальное количество коннектов к БД)

### DISCUSSION THREAD

**Discussion by eodhun** [totumonline/totum-ru-issues-and-discussions#10] [Как это сделать?] [answered]
Добрый день, у нас в базу на Тотум льются данные через remouts в большом объеме и переодически теряется связь с базой, либо появляется ошибка: "Ошибка подключения к базе данных . Попробуйте позже:SQLSTATE[08006] [7] FATAL: remaining connection slots are reserved for non-replication superuser connections"
Пример на скрине: 

![photo_2021-10-04_17-44-39](https://user-images.githubusercontent.com/73879447/135853703-ec0932e6-2fdf-4483-b093-f0110d014920.jpg)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Добрй день. Дело в том что параллельно запускается слишком много remouts. Нужно либо переписать и лить пачками, либо увеличить количество max_connections у базы.

По умолчанию у postgresql это значение равно 100.

By default, the max_connections setting within Postgres is set to 100 connections.

SELECT name, current_setting(name) 
FROM pg_settings 
WHERE name = 'max_connections'

Что бы увеличить надо поменять и рестартовать сервер.

https://stackoverflow.com/questions/36085499/how-to-change-max-connections-for-postgres-through-sql-command
